templates/clair/clair-dpl.yaml

```diff
@@ -53,16 +53,41 @@ spec:
         - name: NO_PROXY
           value: "{{ template "harbor.noProxy" . }}"
         {{- end }}
-{{- if .Values.clair.resources }}
+{{- if .Values.clair.clair.resources }}
         resources:
-{{ toYaml .Values.clair.resources | indent 10 }}
+{{ toYaml .Values.clair.clair.resources | indent 10 }}
 {{- end }}
         ports:
         - containerPort: 6060
         volumeMounts:
         - name: config
           mountPath: /etc/clair/config.yaml
           subPath: config.yaml
+      - name: adapter
+        image: {{ .Values.clair.adapter.image.repository }}:{{ .Values.clair.adapter.image.tag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        livenessProbe:
+          httpGet:
+            path: /probe/healthy
+            port: 8080
+          initialDelaySeconds: 300
+          periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /probe/ready
+            port: 8080
+          initialDelaySeconds: 30
+          periodSeconds: 10
+        env:
+        - name: SCANNER_CLAIR_URL
+          # To avoid a pod cannot reach itself via service IP when the clusters disable hairpin
```

Comment:
I think it's totally fine as they are always in the same pod and share the same network.